### PR TITLE
Add Wallets

### PR DESCRIPTION
**Pull request by novikovalexandr**

### CODE REVIEW

File: app/src/main/java/com/alexandr4/loftcoin/screens/converter/ConverterFragment.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                .subscribe(event -> viewModel.onSourceAmountChange(event.getEditable().toString()));
+
+        sourceCurrency.setOnClickListener(v -> viewModel.onSourceCurrencyClick());
+        destinationCurrency.setOnClickListener(v -> viewModel.onDestinationCurrencyClick());
+        disposables.add(disposable1);
+    }
+
+    private void initInputs() {
+
+        Disposable disposable1 = viewModel.sourceCurrency().subscribe(s ->
```
Comment (askden8) [inline app/src/main/java/com/alexandr4/loftcoin/screens/converter/ConverterFragment.java:130]: could you name it more descriptive?

File: app/src/main/java/com/alexandr4/loftcoin/screens/rate/CurrencyDialog.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import com.alexandr4.loftcoin.utils.Fiat;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+import androidx.fragment.app.DialogFragment;
+
+public class CurrencyDialog extends DialogFragment {
+
+    public static final String TAG = "CurrencyDialog";
```
Comment (askden8) [inline app/src/main/java/com/alexandr4/loftcoin/screens/rate/CurrencyDialog.java:18]: where do you use this?

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsFragment.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        toolbar.setTitle(R.string.wallets_screen_title);
+        toolbar.inflateMenu(R.menu.menu_wallets);
+
+        int screenWidth = getScreenWidth();
+        int walletItemWidth = getResources().getDimensionPixelOffset(R.dimen.item_wallet_width);
+        int walletItemMargin = getResources().getDimensionPixelOffset(R.dimen.item_wallet_margin);
+        int pageMargin = (screenWidth - walletItemWidth) - walletItemMargin;
+
+        walletsPager.setPageMargin(-pageMargin);
+        walletsPager.setOffscreenPageLimit(5);
```
Comment (askden8) [inline app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsFragment.java:81]: pls don't use "magic numbers"

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    public boolean isViewFromObject(@NonNull View view, @NonNull Object object) {
+        return view == object;
+    }
+
+    static class WalletViewHolder extends RecyclerView.ViewHolder {
+
+        @BindView(R.id.symbol_text)
+        TextView symbolText;
+
+
```
Comment (askden8) [inline app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java:77]: many spaces

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+        TextView primaryAmount;
+
+        @BindView(R.id.secondary_amount)
+        TextView secondaryAmount;
+
+        private Random random = new Random();
+
+        private CurrencyFormatter currencyFormatter = new CurrencyFormatter();
+
+        private Context context;
```
Comment (askden8) [inline app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java:91]: why you need this?

File: app/src/main/res/layout/bottom_sheet_currencies.xml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        android:text="@string/currencies_bottom_sheet_title"
+        android:textColor="@color/white"
+        android:textSize="16sp" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+
```
Comment (askden8) [inline app/src/main/res/layout/bottom_sheet_currencies.xml:24]: spaces

Comment (novikovalexandr) [reply app/src/main/res/layout/bottom_sheet_currencies.xml:24]: changed

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java (hunk 5 above)
Comment (novikovalexandr) [reply app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java:91]: changed

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java (hunk 4 above)
Comment (novikovalexandr) [reply app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsPagerAdapter.java:77]: changed

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsFragment.java (hunk 3 above)
Comment (novikovalexandr) [reply app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsFragment.java:81]: changed

File: app/src/main/java/com/alexandr4/loftcoin/screens/converter/ConverterFragment.java (hunk 1 above)
Comment (novikovalexandr) [reply app/src/main/java/com/alexandr4/loftcoin/screens/converter/ConverterFragment.java:130]: ### disposable1? no. here the character of currency and its text designation for convertible currency is established, but that returns to a variable disposable1 - for me not obviously

File: app/src/main/java/com/alexandr4/loftcoin/screens/rate/CurrencyDialog.java (hunk 2 above)
Comment (novikovalexandr) [reply app/src/main/java/com/alexandr4/loftcoin/screens/rate/CurrencyDialog.java:18]: it is used in RateFragment by a call of a window of dialogue of the choice of currency. for identification of a window of dialogue.

**Pull request merged**